File: sample-center/src/com/bstek/dorado/sample/basic/ViewContextTest.java
package com.bstek.dorado.sample.basic;

import org.springframework.stereotype.Component;

import com.bstek.dorado.annotation.Expose;
import com.bstek.dorado.web.DoradoContext;

@Component
public class ViewContextTest {

	@Expose
	public void testAjax(DoradoContext context) {
		context.setAttribute(DoradoContext.VIEW, "attr1", "value2");

		String attr2 = (String) context.getAttribute(DoradoContext.VIEW,
				"attr2");
		context.setAttribute(DoradoContext.VIEW, "attr2", attr2.toUpperCase());

		context.setAttribute(DoradoContext.VIEW, "attr3",
				"New ViewContext Attribute.");
	}
}

File: sample-center/src/com/bstek/dorado/sample/basic/ViewContextTest.js
function refreshFormViewContext() {
	view.set("#formViewContext.entity", {
		attr1: view.get("context.attr1"),
		attr2: view.get("context.attr2"),
		attr3: view.get("context.attr3")
	});
}

// @Bind view.onReady
!function() {
	view.set("context.attr2", "some text");
	refreshFormViewContext();
}

// @Bind #actionAjax.onSuccess
!function() {
	refreshFormViewContext();
}